webapp/channels/src/components/admin_console/admin_definition.tsx

```diff
@@ -2039,6 +2040,60 @@ const AdminDefinition: AdminDefinitionType = {
                     ],
                 },
             },
+            mobile_security: {
```

Comment:
We should add tests for the admin sections for permissions and licenses check

Reply:
We should, but right now there is no easy way to test this. I will meet with Yasser next week to ramp up on cypress tests and add some tests for that as e2e tests.

We could try to figure out how to properly test this, but I would argue that should go in a different PR, since it may require more effort.

Reply:
Some unit tests won't hurt to at least cover license checks

Reply:
That is the thing. There are no unit tests right now for `admin_definition.ts`. The most similar one are a couple of snapshot tests for `admin_sidebar.tsx` (those are failing now and I will add a couple more to improve this a little bit). The `admin_console.tsx` file also uses the admin definitions, but currently have no tests. So creating new unit tests requires digging into this a bit more (mainly to create these tests "the correct way").

On the other hand, we have several cypress test to deal with different parts of the system console, but only one on playwright, so I wanted to make sure I did things the correct way with playwright with Yasser.